gdk/aws_clients/Greengrassv2Client.py

```diff
@@ -0,0 +1,64 @@
+import logging
+
+
+class Greengrassv2Client:
+    """
+    Greengrasv2 client utils wrapper
+    """
+
+    def __init__(self, project_configuration, service_clients):
+        self.project_config = project_configuration
+        self.greengrass_client = service_clients["greengrass_client"]
+
+    def get_highest_component_version_(self) -> str:
+        """
+        Gets highest version of the component from the sorted order of its versions from an account in a region.
+
+        Makes a boto3 request with greengrass service client to list all versions of a component in an account.
+
+        Returns
+        ----
+            version: Highest version of the component if it exists already. Else returns None.
+        """
+
+        try:
+            region = self.project_config["region"]
```

Comment:
Agreed but I'm deferring this as we will be reading project configuration from a new `ProjectConfiguration` object instead of project_config dict as part of this refactoring.